dash/App/Tokens/AddToken/index.js

```diff
@@ -1,9 +1,40 @@
+import { isValidAddress } from '@ethereumjs/util'
 import React, { Component } from 'react'
 import Restore from 'react-restore'
 import RingIcon from '../../../../resources/Components/RingIcon'
 import link from '../../../../resources/link'
 import svg from '../../../../resources/svg'
 
+const invalidFormatError = 'INVALID CONTRACT ADDRESS'
+const unableToVerifyError = `FAILED TO VERIFY TOKEN'S DATA`
```

Comment:
might want to change this to something more descriptive like "failed to find token by address"

Reply:
maybe "could not find token for address `<address>`"